src/main/java/org/fenixedu/academic/domain/student/Student.java

```diff
@@ -44,11 +44,19 @@
 import org.fenixedu.academic.domain.student.registrationStates.RegistrationState;
 import org.fenixedu.academic.domain.student.registrationStates.RegistrationStateTypeEnum;
 import org.fenixedu.academic.domain.studentCurriculum.ExternalEnrolment;
+import org.fenixedu.academic.dto.student.DomainObjectDeletionBean;
 import org.fenixedu.academic.dto.student.StudentStatuteBean;
 import org.fenixedu.bennu.core.domain.Bennu;
+import org.fenixedu.bennu.core.signals.Signal;
 
 public class Student extends Student_Base {
 
+    public static final String DELETION_BEAN_NUMBER = "number";
+    public static final String DELETION_BEAN_USERNAME = "username";
+    public static final String DELETION_BEAN_NAME = "name";
+
+    public static final String STUDENT_CREATE_SIGNAL = "academic.student.create.signal";
```

Comment:
@joaofreitas-qubit should be STUDENT_DELETE_SIGNAL, correct? Allong  with a value of `academic.student.delete.signal`